settings.gradle

```diff
@@ -27,4 +27,6 @@ include 'core'
 include 'scanner'
 include 'injector'
 include 'library-model-loader'
+include 'checks'
```

Comment:
What's under `:checks`? The only `build.gradle` I see is under `checks/ban-mutable-static`.